### ISSUE TITLE
Could not sync with existing tensorboard event files

### ISSUE THREAD

**Issue by vincentwen1995** [wandb/client#1051]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.36
* Python version: Python 3.7.6
* Operating System: MSYS_NT-10.0-18362

### Description

Describe what you were trying to get done.
Tell us what happened, what went wrong, and what you expected to happen.

I am using Anaconda to manage my Python environment and I installed required dependencies for wandb and also pytorch >1.5.0 and tensorboard = 2.2.1. I followed the instructions to log in and sync using the command line interface, but first it requires me to install tensorflow (according to the documentation it should not be a hard requirement) which I did, but then when I tried to sync, it returns an internal error.

### What I Did

```
wandb sync log_dir

wandb: Syncing log_dir to:
wandb: revived-sun-1 https://app.wandb.ai/username/project_name/runs/2cswv2su
wandb: Found tfevents file, converting...
2020-05-21 00:05:17.302868: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'cudart64_101.dll'; dlerror: cudart64_101.dll not found
2020-05-21 00:05:17.310526: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
WARNING:tensorflow:From h:\anaconda3\envs\conda_env_name\lib\site-packages\tensorflow\python\summary\summary_iterator.py:68: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and:
`tf.data.TFRecordDataset(path)`
Traceback (most recent call last):
  File "h:\anaconda3\envs\conda_env_name\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "h:\anaconda3\envs\conda_env_name\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "H:\Anaconda3\envs\conda_env_name\Scripts\wandb.exe\__main__.py", line 7, in <module>
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\click\decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\wandb\cli.py", line 154, in wrapper
    return func(*args, **kwargs)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\wandb\cli.py", line 447, in sync
    path, run_id=id, project=project, entity=entity, ignore_globs=globs)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\wandb\wandb_run.py", line 339, in from_directory
    summary.update(wbtf.stream_tfevents(path, file_api, run, namespace=namespace))
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\wandb\tensorflow\__init__.py", line 55, in stream_tfevents
    parsed = tf_summary_to_dict(summary, namespace=namespace)
  File "h:\anaconda3\envs\conda_env_name\lib\site-packages\wandb\tensorboard\__init__.py", line 339, in tf_summary_to_dict
    if not wandb.run.config.get(key):
AttributeError: 'NoneType' object has no attribute 'config'
```

**Comment by cvphelps** [comment]
Hi there, thanks for letting us know about this issue. We'd like to fix syncing problems in the new version of our client, and I'll check with my colleague @raubitsj to see if this case is supported.

ccing @raubitsj — we're working on a new beta of the Weights & Biases client that improves support for environments like Windows. Would you be interested in giving it a try? 

1. [Join the Slack forum](http://bit.ly/wandb-forum) and go to the wandb-cling channel.
2. [Install CLI-NG](https://github.com/wandb/client-ng) following directions from the GitHub repo.
3. Try tracking a model with the new client, no code changes necessary.

Message us in the wandb-cling channel— we’d love to hear any questions or feedback.

**Comment by douglasrizzo** [comment]
I also have this problem. My TensorBoard event files were created with PyTorch. I am on Linux.

Manjaro Linux 20.0.3 Lysia
Python 3.7.7
tensorboard 2.2.2
tensorflow 2.2.0
torch 1.5.0

```
wandb: Syncing /run/media/user/657fb98a-8515-4d43-87a4-5000a5c733d6/tensorboard/v7 to:
wandb: visionary-hill-4 https://app.wandb.ai/tetamusha/dodonet/runs/21f1zr3n
wandb: Found tfevents file, converting...
WARNING:tensorflow:From /home/user/.anaconda3/lib/python3.7/site-packages/tensorflow/python/summary/summary_iterator.py:68: tf_record_iterator (from tensorflow.python.lib.io.tf_record) is deprecated and will be removed in a future version.
Instructions for updating:
Use eager execution and:
`tf.data.TFRecordDataset(path)`
Traceback (most recent call last):
  File "/home/user/.anaconda3/bin/wandb", line 8, in <module>
    sys.exit(cli())
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/wandb/cli.py", line 154, in wrapper
    return func(*args, **kwargs)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/wandb/cli.py", line 446, in sync
    path, run_id=id, project=project, entity=entity, ignore_globs=globs)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/wandb/wandb_run.py", line 346, in from_directory
    last_row = wbtf.stream_tfevents(path, file_api, run, step=step, namespace=namespace)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/wandb/tensorflow/__init__.py", line 55, in stream_tfevents
    parsed = tf_summary_to_dict(summary, namespace=namespace)
  File "/home/user/.anaconda3/lib/python3.7/site-packages/wandb/tensorboard/__init__.py", line 343, in tf_summary_to_dict
    if not wandb.run.config.get(key):
AttributeError: 'NoneType' object has no attribute 'config'
```

**Comment by douglasrizzo** [comment]
@vincentwen1995 maybe the discussion in https://github.com/wandb/client-ng/issues/23#issuecomment-651258932 could clarify some things to you.

**Comment by cvphelps** [comment]
Hi @vincentwen1995 I believe this issue is fixed in the latest version of our library, 0.10.0. Could you please try it and reopen the issue if it's still not working?

**Comment by vincentwen1995** [comment]
> Hi @vincentwen1995 I believe this issue is fixed in the latest version of our library, 0.10.0. Could you please try it and reopen the issue if it's still not working?

Sure, thank you!

**Comment by aj7tesh** [comment]
""Could not sync with existing tensorboard event files wandb/client#1051""
Above feature is not yet there,right ?
